### PR TITLE
starting the first part of the 16 chapter

### PR DESCRIPTION
**Pull request by fgazzelloni**

### CODE REVIEW

File: 16-Escaping_the_graph.Rmd [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+  
+  
+## Introduction 
+  
+This chapter based on **Escaping the graph** is made to addressing the reactive programming identifying the connections inside the app, and to use **reactiveVal()** and **observe()/observeEvent()** functions with the scope of applying manual controls over parts of the graph.
+
+To recap: Reactive programming is used to specify a graph of dependencies so that when an input changes, all related outputs are automatically updated. The reactive graph describes how inputs and outputs are connected, it is a powerful tool for understanding how your app works, and it’s useful to sketch the dependencies in your graph.
+
+
+```{r libraries, message=FALSE, warning=FALSE, include=FALSE}
```
Comment (russHyde) [inline 16-Escaping_the_graph.Rmd]: just remove the chunk name, and the pipeline should get a bit further.
That is, replace with ```{r, message = FALSE, warning = FALSE, include = FALSE}

**Comment by fgazzelloni** [comment]
Hello, 
I have added chapter 16
thanks!

**Review by russHyde — APPROVED** [review]
LGTM

**Comment by fgazzelloni** [comment]
updating the app  with eval = FALSE, echo = TRUE

**Pull request merged**